README-HI.md

```diff
@@ -0,0 +1,141 @@
+        <li><a href="./README-JA.md"> 日本語 </a></li>
+        <li><a href="./README-HI.md"> हिंदी </a></li>
+    </ul>
+</details>
+<!-- Do not translate this -->
+
+# आपका स्वागत है नौसिखिया ओपन सोर्स योगदानकर्ताओं!
+
+[![अनुरोध का स्वागत करें](https://img.shields.io/badge/PRs-welcome-brightgreen.svg?style=flat)](http://makeapullrequest.com)
+[![पहली बार केवल अनुकूल](https://img.shields.io/badge/first--timers--only-friendly-blue.svg)](https://www.firsttimersonly.com/)
+[![स्थिति बनाएँ](https://api.travis-ci.org/freeCodeCamp/how-to-contribute-to-open-source.svg?branch=master)](https://travis-ci.org/freeCodeCamp/how-to-contribute-to-open-source)
+
+
+यह उन लोगों के लिए संसाधनों की एक सूची है जो ओपन सोर्स में योगदान करने के लिए नए हैं।
+
+यदि आपको अतिरिक्त संसाधन मिलते हैं, तो कृपया एक पुल अनुरोध में योगदान करें।
+
+यदि आपके कोई प्रश्न या टिप्पणी है, तो कृपया एक मुद्दा बनाएं।
+
+**अंतर्वस्तु**
+
+- [सामान्य रूप से ओपन सोर्स में योगदान करना](#contributing-to-open-source-in-general)
+- [डायरेक्ट गिटहब खोजता है](#direct-github-searches)
+- [मोज़िला का योगदानकर्ता पारिस्थितिकी तंत्र](#mozillas-contributor-ecosystem)
+- [नए ओपन सोर्स योगदानकर्ताओं के लिए उपयोगी लेख](#useful-articles-for-new-open-source-contributors)
+- [संस्करण नियंत्रण का उपयोग करना](#using-version-control)
+- [स्रोत पुस्तकें खोलें](#open-source-books)
+- [ओपन सोर्स योगदान पहल](#open-source-contribution-initiatives)
+- [लाइसेंस](#license)
+
+## सामान्य रूप से ओपन सोर्स में योगदान करना
+- [ओपन सोर्स में योगदान करने के लिए निश्चित गाइड](https://www.freecodecamp.org/news/the-definitive-guide-to-contributing-to-open-source-900d5f9f2282/) by [@DoomHammerNG](https://twitter.com/DoomHammerNG)
+- [ओपन सोर्स के लिए एक परिचय](https://www.digitalocean.com/community/tutorial_series/an-introduction-to-open-source) - GitHub पर सफलता में योगदान करने के लिए अपने रास्ते पर मार्गदर्शन करने के लिए DigitalOcean द्वारा ट्यूटोरियल।
+- [Issuehub.io](http://issuehub.io/) - लेबल और भाषा द्वारा GitHub मुद्दों को खोजने के लिए एक उपकरण।
+- [Code Triage](https://www.codetriage.com/) - अन्य, वास्तव में अच्छा, लोकप्रिय रिपॉजिटरी और भाषा द्वारा फ़िल्टर किए गए मुद्दों को खोजने के लिए उपकरण।
+- [Awesome-for-beginners](https://github.com/MunGell/awesome-for-beginners) - एक GitHub रेपो जो नए योगदानकर्ताओं के लिए अच्छे बग्स के साथ प्रोजेक्ट करता है, और उनका वर्णन करने के लिए लेबल लागू करता है।
+- [Open Source Guides](https://opensource.guide/) - उन व्यक्तियों, समुदायों और कंपनियों के लिए संसाधनों का संग्रह, जो ओपन सोर्स प्रोजेक्ट को चलाना और उसमें योगदान देना सीखना चाहते हैं।
+- [45 Github Issues Dos and Don’ts](https://hackernoon.com/45-github-issues-dos-and-donts-dfec9ab4b612) -GitHub पर Do's और Don'ts।
+- [GitHub Guides](https://guides.github.com/) - कैसे प्रभावी रूप से GitHub का उपयोग करने के बारे में बुनियादी गाइड।
+- [Contribute to Open Source](https://github.com/danthareja/contribute-to-open-source) - एक सिमुलेशन परियोजना के लिए कोड का योगदान करके GitHub वर्कफ़्लो जानें।
```

Comment:
- Corrected transliteration of 'Simulation'

- Better not to translate the word 'project'

```suggestion
- [Contribute to Open Source](https://github.com/danthareja/contribute-to-open-source) - एक सिम्युलेशन प्रोजेक्ट के लिए कोड का योगदान करके GitHub वर्कफ़्लो जानें।
```